scilifelab/report/survey.py

```diff
@@ -61,6 +61,17 @@ def project_closed(project, format="%Y-%m-%d"):
     except:
         pass
     return date
+
+def project_aborted(project):
```

Comment:
just a suggestion, but usually, when it is a checking function, one usually names it `is_*`, so in this case would be `is_project_aborted` or just `is_aborted`